package.json

```diff
@@ -84,5 +87,10 @@
   "bugs": {
     "url": "https://github.com/MRN-Code/nodeapi/issues"
   },
-  "homepage": "https://github.com/MRN-Code/nodeapi"
+  "homepage": "https://github.com/MRN-Code/nodeapi",
+  "pre-commit": [
+    "lint",
+    "validate",
+    "test"
+  ]
```

Comment:
hopefully you guys are OK with doing these things on commit. there's always `-n`, but noone has to know :)